raiden/transfer/state.py

```diff
@@ -1368,8 +1368,15 @@ def __init__(self, address: Address, balance: Balance):
         self.address = address
         self.contract_balance = balance
 
+        #: Locks which have been introduced with a locked transfer, however the
```

Comment:
What is the colon for?

Reply:
It's for documentation using sphinx.

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Reply:
Ah nice, didn't know that. But then you forgot to add them to the comment of `secrethashes_to_onchain_unlockedlocks`

Reply:
done